crates/bitcoin-da/src/helpers/mod.rs

```diff
@@ -22,6 +24,7 @@ enum TransactionKindLightClient {
     Unknown(NonZeroU16),
 }
 
+#[cfg(feature = "native")]
```

Comment:
i wouldn't call these native, since these are gonna be used in a zk circuit (outside of our repo)